close.in.sample

```diff
@@ -7,7 +7,7 @@
  ce.out
 )---------------------------------------------------------------------
  express time in days or years = years
- express time relative to integration start time = yes
+ express time relative to integration start time = no
```

Comment:
I used this value to make easier comparing with list of close approaches (e. g. for Apophis), where both Julian and Gregorian dates are given.
http://ssd.jpl.nasa.gov/sbdb.cgi?sstr=apophis;old=0;orb=0;cov=0;log=0;cad=1#cad
As described in mercury6 manual, that value and some others in ".in" files can be changed by user if he wants to specify the output format